.github/workflows/tests-clustermesh-upgrade.yaml

```diff
@@ -135,6 +135,13 @@ jobs:
           image-tag: ${{ inputs.SHA }}
           chart-dir: ./untrusted/cilium-newest/install/kubernetes/cilium
 
+      - name: Get Cilium's default values
+        id: downgrade-default-vars
+        uses: ./.github/actions/helm-default
+        with:
+          image-tag: ${{ inputs.SHA }}
+          chart-dir: ./untrusted/cilium-downgrade/install/kubernetes/cilium
```

Comment:
I'd move this below the `downgrade-vars` step, to directly configure the correct image tag (i.e., the one retrieved there). At that point, you could also drop the `CILIUM_IMAGE_SETTINGS` part from that step (and possibly rename the step), as that would already be covered by the `helm-default` action. Finally, you could also drop both `--set=debug.enabled=true` and `--set=bpf.monitorAggregation=none` from the `vars` step below, as it would be also covered by the `helm-default` action.